helm-repositories/banzaicloud/banzaicloud.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kubernetes-charts.banzaicloud.com
+spec:
+  url: https://kubernetes-charts.banzaicloud.com
+  interval: 10m
```

Comment:
minor -- add a new line here

Reply:
fixed in a705b0a164db719303729f66088558088d40cd48!